openlibrary/core/wikidata.py

```diff
@@ -19,6 +19,63 @@
 WIKIDATA_API_URL = 'https://www.wikidata.org/w/rest.php/wikibase/v0/entities/items/'
 WIKIDATA_CACHE_TTL_DAYS = 30
 
+SOCIAL_PROFILES = [
```

Comment:
I decided not to move this to a separate file for now because we only have one. Likewise with the dataclass. I think we can do that when we add more profiles if it looks better that way.